modules/nested-accordion/assets/scss/frontend.scss

```diff
@@ -34,21 +33,27 @@
 				display: flex;
 				flex-direction: row;
 				list-style: none;
-				font-size: var(--n-accordion-title-font-size);
 			  	padding: var(--n-accordion-padding);
 			  	border-width: var(--n-accordion-border-width);
 				border-color: var(--n-accordion-border-color);
 				border-style: var(--n-accordion-border-style);
-				justify-content: var(--n-accordion-item-title-justify-content);
+				justify-content: var(--n-accordion-title-justify-content);
 				border-radius: var(--n-accordion-border-radius);
-			  	flex-grow: var(--n-menu-item-title-flex-grow);
+				flex-grow: var(--n-menu-title-flex-grow);
+
+				&-text h1,h2,h3,h4,h5,h6,p {
+					margin-block: 0;
+				}
 
 				&-icon {
 					display: flex;
 					flex-direction: row;
 					align-items: center;
 					padding-inline: var( --n-accordion-title-icon-padding-inline );
 					order: var( --n-accordion-title-icon-order );
+					overflow: hidden;
```

Comment:
@Barcov Why do we need `overflow:hidden`?

Reply:
Agreed